stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/plugin/logic/querychannels/internal/QueryChannelsLogic.kt

```diff
@@ -35,7 +35,7 @@ import io.getstream.chat.android.offline.event.handler.chat.EventHandlingResult
 import io.getstream.logging.StreamLog
 import kotlinx.coroutines.flow.StateFlow
 
-private const val MESSAGE_LIMIT = 30
+private const val MESSAGE_LIMIT = 1
```

Comment:
Hey @leandroBorgesFerreira,

I'm wondering how the `MESSAGE_LIMIT ` is connected to multiple channel queries?

Reply:
😱

Reply:
It is not, this PR doesn't fix the [ISSUE-4230](https://github.com/GetStream/stream-chat-android/issues/4230). This is just a small correction in the default message limit of channel request queries.